src/App.tsx

```diff
@@ -1,24 +1,58 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/exhaustive-deps */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { USER_ID } from './utils/constants';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
+import { ErrorMessage } from './types/ErrorMessage';
+import { TodoContext } from './contexts/TodoContext';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
+  const {
+    setErrorMessage,
+    todos,
+    setTodos,
+  } = useContext(TodoContext);
+
+  const handleOverLoad = () => {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  };
+
+  useEffect(() => {
+    getTodos(84)
```

Comment:
please do updates according to comments from previous task

https://github.com/mate-academy/react_todo-app-loading-todos/pull/1115#pullrequestreview-1862804333